otk8slibs/k8s_kube_session.py

```diff
@@ -0,0 +1,9 @@
+from kubernetes import  config
+
+def _create_k8s_session(k8s_context=None):
+    
+    if k8s_context:
+        config.load_kube_config(context=k8s_context)
+    else:
+        config.load_kube_config(context="minikube")
```

Comment:
Do we need to mention "minikube"  Please explore what context it will pick by default if we dont give any context.